yb_migrate/cmd/importData.go

```diff
@@ -858,10 +876,10 @@ func doOneImport(t *SplitFileImportTask, targetChan chan *tgtdb.Target) {
 				utils.ErrExit("rename %q => %q: %s", inProgressFilePath, doneFilePath, err)
 			}
 
-			err = os.Truncate(doneFilePath, 0)
-			if err != nil {
-				log.Warnf("truncate file %q: %s", doneFilePath, err)
-			}
+			// err = os.Truncate(doneFilePath, 0)
+			// if err != nil {
+			// 	log.Warnf("truncate file %q: %s", doneFilePath, err)
+			// }
```

Comment:
Why?

Reply:
I commented this for testing and accidentally pushed this part.
Thank you very much for pointing it out!!